src/main/resources/webapp/js/components/ClustersComponent.js

```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+
+var ClustersComponent = React.createClass({
+  displayName: "ClustersComponent",
+
+  render: function () {
+    return(
+      <h1>Clusters</h1>
+   )}
+
+});
+
+module.exports = ClustersComponent;
```

Comment:
I agree with @DarinJ.  I think I understand where this is going, however it is a potential future.   Currently we only support 1 cluster per framework.   unless I'm missing something... lets remove.